scania/corporate-ui-site
# Improvement/update header visual

**Describe pull-request**  
Add header details to the site. 
Mobile first SCSS

**Solving issue**  

Fixes: #112 

**How to test**  
1. Check on mobile and desktop
2. Check requirements in https://github.com/scania/corporate-ui-site/issues/112
3. Compare to Figma >> UI Kit >> Header

**Screenshots**  
Mobile
![image](https://user-images.githubusercontent.com/1199101/100724195-e6930980-33c2-11eb-9ca3-3e0666c387ba.png)
Mobile search open
![image](https://user-images.githubusercontent.com/1199101/100724222-eeeb4480-33c2-11eb-9bb5-dc7413e0df9f.png)
Desktop search open 
![image](https://user-images.githubusercontent.com/1199101/100724265-fad70680-33c2-11eb-880f-be33b3080d60.png)
Desktop
![image](https://user-images.githubusercontent.com/1199101/100724314-062a3200-33c3-11eb-93ab-55d5dceaffea.png)


**Additional context**  
_Add any other context about the pull-request here._
